zap/src/main/java/org/zaproxy/zap/extension/alert/AlertAPI.java

```diff
@@ -474,6 +474,7 @@ private void processAlerts(
         Map<String, String> map = new HashMap<>();
         map.put("id", String.valueOf(alert.getAlertId()));
         map.put("pluginId", String.valueOf(alert.getPluginId()));
+        map.put("alertRef", alert.getAlertRef());
```

Comment:
Should we also update the core reporting? (Or separate issue/PR?)

Reply:
Doh, forgot about those, will add to this PR...

Reply:
Ha! No need - they now call the alert API endpoints so they dont need to change.